### DISCUSSION TITLE
Compare to React-Admin?

### DISCUSSION THREAD

**Discussion by osseonews** [refinedev/refine#1917] [General]
We use React-Admin on many projects and I'm curious how Refine is different ? Other than using Ant vs Material U, the frameworks seem basically the same, dataProvider, authProvider etc. What are the differences and how is Refine possibly improving on React-Admin? Thanks.

**Comment by omeraplak** [comment]
Hey @osseonews ,
Thank you for contacting us! 

Although the two projects are similar in approach, Refine solves the problem with a very different approach.

Since Refine is headless, it is completely independent of the UI. Refine has native Ant Design support, we will soon add our Material UI support, then Mantine, etc. So Refine will support all popular UI libraries.

Refine's way of doing this is purely by using React's hooks architecture. Unlike this, React-Admin prefers Component architecture (as we see they are trying to switch to hook architecture as well).

Finally, Refine is completely Open-Source. now and forever. All the features that React-Admin offers in the Enterprise package (RBAC, Live/Realtime, Audit log, etc.) are available in Refine. and Open-Source.

For a more detailed comparison, please visit our Comparision page,
https://refine.dev/docs/comparison/

Finally, We love React Admin and appreciate all the effort they've made so far.

**Reply by osseonews** [reply]
Thank you! Yes, I didn't notice that you already have a comparison page. Super helpful. I did notice that alot of what you offer out of the box is only enterprise in React-admin, so this makes a big difference. Anyway, we'll try Refine out for our next project. Thanks for your help.

**Reply by omeraplak** [reply]
Please don't forget to share your experiences with the community 🎉
